config/routes.rb

```diff
@@ -22,6 +22,7 @@
 
     scope :application do
       get '/' => 'application_form#show', as: :application_form
+      get '/application/review' => 'application_review#show'
```

Comment:
> URL for review:
> 
> <img alt="Screenshot 2019-10-14 at 17 07 48" width="607" src="https://user-images.githubusercontent.com/58971/66766148-55eb2880-eea5-11e9-8b34-c31255d71a6c.png">

Are we sure we want this as the URL with the repetition of `/application`?

Reply:
of course not ;) I'll fix it, thanks!